src/main/java/org/opentripplanner/model/plan/legreference/LegReferenceSerializer.java

```diff
@@ -26,7 +28,7 @@ private LegReferenceSerializer() {}
   @Nullable
   public static String encode(LegReference legReference) {
     if (legReference == null) {
-      return null;
+      return notAvailable;
```

Comment:
Since u change this, do you also need to update the transmodel API handling of missing value?

Reply:
Transmodel API works as expected. null is encoded as 'NotAvailable' and 'NotAvailable' is decoded as null.

The only change I can think of is maybe removing isBlank() check from TransmodelGraphQLSchema.java.